src/main/webapp/WEB-INF/tags/navBarBrand.tag

```diff
@@ -0,0 +1,8 @@
+<%@ tag description="TEAMMATES logo brand"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ tag import="teammates.common.util.Const" %>
```

Comment:
Hm, you're absolutely right. I have ran the tests after removing the unnecessary lines and found that it works correctly, so I will be removing them!